testing/test_assertion.py

```diff
@@ -981,6 +981,36 @@ class SimpleDataObjectTwo:
         assert lines is None
 
 
+class TestAssert_reprcompare_namedtuple:
+    def test_namedtuple(self) -> None:
+        SimpleDataObject = collections.namedtuple(
+            "SimpleDataObject", ["field_a", "field_b"]
+        )
+
+        left = SimpleDataObject(1, "b")
+        right = SimpleDataObject(1, "c")
+
+        lines = callequal(left, right)
+        assert lines is not None
+        assert lines[2].startswith("Omitting 1 identical item")
+        assert "Matching attributes" not in lines
+        for line in lines[2:]:
+            assert "field_a" not in line
+
```

Comment:
Please also add a test which tests that a comparison between a `namedtuple` and a plain tuple works as intended.